nemo_skills/pipeline/utils.py

```diff
@@ -498,6 +498,13 @@ def get_env_variables(cluster_config):
         elif env_var in default_factories:
             env_vars[env_var] = default_factories[env_var]()
             logging.info(f"Adding optional environment variable {env_var} from environment")
+        elif "=" in env_var:
```

Comment:
I'd say this case should take precedence over the environment one. So, if you have it explicitly in the config and also in the environment, we should take the one from the config, but right now it seems that environment one will be used. Also maybe change the logging message to add "with value from the config" or something like that to make it easier to see

Reply:
Hmm it seems to work locally, but moved it up before env one just in case.